### PR TITLE
[balance-of-with-thresholds] Add balance-of-with-thresholds strategy

### PR DESCRIPTION
**Pull request by lucid-eleven**
Changes proposed in this pull request:
- Add a new strategy for ERC20 or ERC721, allocates votes based on balance mapped against a table of minimum thresholds

### CODE REVIEW

**Review by gawainb — APPROVED** [review]
Great strategy implementation. We also have a request for the strategy.

**Comment by lucid-eleven** [comment]
thanks @ChaituVR, anything else I need to do to get this merged in?

File: src/strategies/balance-of-with-thresholds/index.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+
+export const author = 'eleven';
```
Comment (ChaituVR) [inline src/strategies/balance-of-with-thresholds/index.ts:4]: Hi, this should be your github username
```suggestion
export const author = 'lucid-eleven';
```

**Review by bonustrack — APPROVED** [review]
Looks good!

**Pull request merged**